File: Domain/Hero.cs
namespace Domain;

public class Hero
{
    //public virtual ICollection<Game>? Games { get; set; }
    public int Id { get; set; }
    public string Name { get; set; }
    public int ContentId { get; set; }
    public virtual  Content? Content { get; set; }
    public string Image { get; set; }
}
    

File: Angular/src/app/hero.ts
import { Content } from "./content";

export class Hero {
  public id : number;
  public name : string;
  public contentId : string;
  public image : string;
  public content : Content;

  constructor() {
    this.id = -1;
    this.name = "";
    this.contentId = "";
    this.image = "";
    this.content = new Content();
  }
}
